modules/node_groups/node_groups.tf

```diff
@@ -57,6 +57,16 @@ resource "aws_eks_node_group" "workers" {
     }
   }
 
+  dynamic "taint" {
+    for_each = each.value["taints"] == null ? [] : each.value["taints"]
```

Comment:
Can we just define `taints = []` default value in https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/node_groups/locals.tf#L5-L23

Reply:
Why? I think no one will to define taints in workers_group_defaults to apply taints to all node_groups.

Reply:
Sorry, I misunderstood the role of the node_groups_expanded,  Thanks for the suggestion, I'll change it.

Reply:
> Why? I think no one will define taints in workers_group_defaults to apply taints to all node_groups

You're absolutely right. It's just for consistency with other arguments. That local, is used most of the time to know what are available arguments. And it's also for simplicity.

Reply:
force pushed